app/src/ui/navbar/account-widget/AccountWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import clsx from "clsx";
+import { AccountWidgetProps } from "./AccountWidget.types";
+import styles from "./AccountWidget.module.scss";
+import { Dropdown } from "ui/dropdown/Dropdown";
+import { Icon } from "ui/icon/Icon";
+import { Typography } from "ui/typography/Typography";
+import { useToggle } from "hooks/useToggle/useToggle";
+import { useTimeout } from "hooks/useTimeout/useTimeout";
+import { useAuthContext } from "hooks/useAuthContext/useAuthContext";
+import { Session } from "@supabase/gotrue-js";
+
+export const AccountWidget: React.FC<AccountWidgetProps> = ({ className }) => {
+  const { isOpen, close: closeMenu, open: openMenu } = useToggle(false);
+  const { session } = useAuthContext();
+
+  const getUserNickname = (currentSession: Session | null) => {
+    const userEmail = currentSession?.user?.email;
+    const userNickname = userEmail?.slice(0, 2).toUpperCase();
+
+    return userNickname;
+  };
+
+  useTimeout(
+    () => {
+      closeMenu();
+    },
+    [isOpen],
+    6000,
+  );
```

Comment:
`useTimeout` seems smelly: 

1. We can simply use: `setTimeout(() => {}, 6000)`, no need to wrap it in a hook, no need for a timeout at all though, because:
2. Closing the dropdown with a timeout is not a common use-case, what you really need to do is create a backdrop. This is the way other projects do it: https://github.com/bancosatoshi/web/blob/a43230b1658a7c40c539ab7847c3cce8de6b4752/app/src/ui/wallet-selector/WalletSelector.tsx#L42

```suggestion
  useTimeout(
    () => {
      closeMenu();
    },
    [isOpen],
    6000,
  );
```